spec/rubocop/cop/style/string_literals_in_interpolation_spec.rb

```diff
@@ -8,10 +8,13 @@
       expect_offense(<<~'RUBY')
         "#{"A"}"
            ^^^ Prefer single-quoted strings inside interpolations.
+        :"#{"A"}"
+            ^^^ Prefer single-quoted strings inside interpolations.
```

Comment:
Can you add a new test case without directly editing the existing test case?

Reply:
Done.